txselector/txselector_test.go

```diff
@@ -411,123 +411,6 @@ func TestGetL2TxSelectionMinimumFlow0(t *testing.T) {
 	txsel.LocalAccountsDB().Close()
 }
 
-func TestPoolL2TxsWithoutEnoughBalance(t *testing.T) {
```

Comment:
deleted this test, because we don't check the balance in splitL2ForgableAndNonForgable